<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesFormGeneral</class>
 <widget class="QWidget" name="PropertiesFormGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_icon">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="label_display">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_name_text">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="label_name">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_size_text">
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLabel" name="label_size">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_path_text">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLabel" name="label_path">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="buttonOpenPath">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/images/file-open_container_folder.png</normaloff>:/images/file-open_container_folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_type_text">
     <property name="text">
      <string>Type of file:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLabel" name="label_type">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_time_creation_text">
     <property name="text">
      <string>Created:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QLabel" name="label_time_creation">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_time_modified_text">
     <property name="text">
      <string>Last modified:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QLabel" name="label_time_modified">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_time_lastaccess_text">
     <property name="text">
      <string>Last access:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="QLabel" name="label_time_lastaccess">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_attributes_text">
     <property name="text">
      <string>Attributes:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="checkbox_hidden">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Hidden</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QCheckBox" name="checkbox_system">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>System</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QCheckBox" name="checkbox_archive">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Archive</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QCheckBox" name="checkbox_readonly">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Read only</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOpenPath</sender>
   <signal>clicked()</signal>
   <receiver>PropertiesFormGeneral</receiver>
   <slot>slotOpenPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOpenPath()</slot>
 </slots>
</ui>
